web/src/components/TestConnectionNotification/TestConnectionNotification.styled.ts

```diff
@@ -1,4 +1,4 @@
-import {CheckCircleFilled, MinusCircleFilled} from '@ant-design/icons';
+import {CheckCircleFilled, MinusCircleFilled, WarningFilled} from '@ant-design/icons';
```

Comment:
I used the warning icon, but it doesn't look great. Any suggestion of an icon that we can use? Especially because there's no `WarningCircleFilled` icon :disappointed:

Reply:
@mathnogueira for keeping consistency we can use round warning icon: 
<img width="793" alt="Screenshot 2023-02-28 at 1 41 38 PM" src="https://user-images.githubusercontent.com/5725840/221961486-f740a5f9-c814-4fd8-984f-801d4a77f49e.png">
i will add here svg for it:
![ExclamationCircle](https://user-images.githubusercontent.com/5725840/221961580-a71aec2a-57fd-4ab3-8ee0-52c646428603.svg)